### PR TITLE
TonY release pipeline

### PR DESCRIPTION
**Pull request by hungj**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,5 +5,5 @@ jdk:
   - oraclejdk8
 env:
   - HADOOP_VERSION=3.1.1
-script: "./gradlew test --stacktrace --info"
+script: "./gradlew test --stacktrace --info && ./gradlew ciPerformRelease"
```
Comment (oliverhu) [inline .travis.yml]: this will be called for every travis job - every PR/every branch

File: build.gradle [hunk 2]
```diff
@@ -17,9 +11,17 @@ buildscript {
     classpath "com.github.jengelman.gradle.plugins:shadow:2.0.4"
     classpath "com.google.protobuf:protobuf-gradle-plugin:0.8.1"
     classpath "gradle.plugin.nl.javadude.gradle.plugins:license-gradle-plugin:0.14.0"
+    classpath "org.shipkit:shipkit:2.0.13"
```
Comment (oliverhu) [inline build.gradle]: what doe this do?

File: .travis.yml (hunk 1 above)
Comment (hungj) [reply .travis.yml]: should be fine. shipkit will not publish if it is a PR

File: build.gradle (hunk 2 above)
Comment (hungj) [reply build.gradle]: It's the dependency which handles pushing to bintray which we use to sync to maven central

File: gradle/shipkit.gradle [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  gitHub.writeAuthToken = System.getenv("GH_WRITE_TOKEN")
+}
+
+allprojects {
+  plugins.withId("org.shipkit.bintray") {
+
+    //Bintray configuration is handled by JFrog Bintray Gradle Plugin
+    //For reference see the official documentation: https://github.com/bintray/gradle-bintray-plugin
+    bintray {
+
```
Comment (oliverhu) [inline gradle/shipkit.gradle]: additional blank line

File: version.properties [hunk 4]
```diff
@@ -0,0 +1,3 @@
+#Version of the produced binaries. This file is intended to be checked-in.
```
Comment (oliverhu) [inline version.properties]: space after # , same next line

**Pull request merged**